torchbenchmark/models/hf_GPTJ/__init__.py

```diff
@@ -0,0 +1,10 @@
+from torchbenchmark.tasks import NLP
+from torchbenchmark.util.framework.huggingface.model_factory import HuggingFaceModel
+
+class Model(HuggingFaceModel):
+    task = NLP.LANGUAGE_MODELING
+    DEFAULT_TRAIN_BSIZE = 1
```

Comment:
Is this the correct batch size? We should use the default batch size in the upstream code if possible.